tesseract_ros/tesseract_msgs/msg/Trajectory.msg

```diff
@@ -1,5 +1,10 @@
 TesseractState tesseract_state
 
+# Tesseract Command Language Instructions
+string instructions
+
+# DEPRECATED should use instructions
```

Comment:
I'm not sure about this. A JointTrajectory is pretty standard. I suppose as long as the plotter has necessary utilities to convert a JointTrajectory to command language that is fine.

Reply:
I am fine with leaving both. I will remove the deprecated.

Reply:
removed.